### PR TITLE
Updated participant input from <select> to ("type", "number") <input>

### PR DESCRIPTION
**Pull request by alfredbrannare**
Updated the <select> input displaying participants to a type = number <input>. I also added a label displaying the min and max participant range so the user knows what the alternatives are. Will check on alternative warning text later.

https://github.com/user-attachments/assets/0bb04d22-b35a-4fac-a0fa-f6ac27136bb1

### CODE REVIEW

**Review by Paajt — APPROVED** [review]
LG2M, approved!

File: modal.js [hunk 1]
```diff
@@ -138,22 +138,16 @@ function openBookingWindow(challengeTitle, challengeId, availableTimes = [], par
...
+        const amountOfPeopleInput = document.createElement("input");
+        amountOfPeopleInput.classList.add("amountOfPeopleInput");
+        amountOfPeopleInput.setAttribute("type", "number");
+
+        const participantMin = Math.min(...participantOptions);
+        const participantMax = Math.max(...participantOptions);
+
+        const amountOfPeopleRange = document.createElement("label");
+        amountOfPeopleRange.classList.add("amountOfPeopleRange");
+        amountOfPeopleRange.innerHTML = `Allowed participants ${participantMin} - ${participantMax}`;
```
Comment (Emilyempa) [inline modal.js:150]: I like the fact that u remind user how many participants is allowed in room

**Review by Emilyempa — APPROVED** [review]
Nice work! Ok to merge for me

**Review by itsJoel59 — APPROVED** [review]
I agree with Emily and Patrik, ok to merge! Nice work Alfred

**Review by NordicNomadicLife — APPROVED** [review]
nicely done, i agree with the rest and okey to merge for me

**Pull request merged**